frontend/src/utilities/useWatchNotebook.tsx

```diff
@@ -66,7 +66,7 @@ export const useWatchNotebook = (
     };
     // Don't update when components are updated
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [username, pollInterval]);
+  }, [username, pollInterval, projectName]);
```

Comment:
This is a disconcerting change -- this is why we want linters in place and not to ignore them.

I'll look at making this change in my Admin PR as I need to watch multiple notebooks so I'll need to rework this hook too.

Reply:
Updated.